prow/cmd/deck/main.go

```diff
@@ -419,9 +419,13 @@ func main() {
 		}
 	}
 
-	authCfgGetter := func(refs *prowapi.Refs) *prowapi.RerunAuthConfig {
-		rac := cfg().Deck.RerunAuthConfigs.GetRerunAuthConfig(refs)
-		return &rac
+	authCfgGetter := func(refs *prowapi.Refs, cluster string) *prowapi.RerunAuthConfig {
+		err := cfg().Deck.FinalizeDefaultRerunAuthConfigs()
```

Comment:
Why is it still done here? It is basically impossible it will fail here because we already executed it during config loading and if it failed there, we won't even start so doing it again here is pointless or am I missing something?